### DISCUSSION TITLE
what should i use instead of FOSUserBundle?

### DISCUSSION THREAD

**Discussion by AndresMonroy02** [symfony/symfony#50239] [Q&A] [answered]
I am upgrading my Symfony 3.4 project to Symfony 5.4 , the previous developer used FosUserBundle for user management, now it is deprecated, should I use another bundle or create a new login using symfony documentation:https://symfony.com/doc/current/security.html#the-user

if it is another bundle wich one?

Thanks

**Comment by stof** [comment, marked as answer]
Well, the readme of the bundle provides a good summary of the recommended replacement features: https://github.com/FriendsOfSymfony/FOSUserBundle#maintenance-status

Note however that you don't need to to that work as part of your upgrade of Symfony. FOSUserBundle is not deprecated yet. I still maintain it so that existing projects depending on it can keep upgrading Symfony (my own project at work is in that situation as well actually).
So my advice is to first perform the upgrade of Symfony while keeping the bundle, and then working on replacing the bundle features in a separate step.
